docs/faqs/faqs.md

```diff
@@ -0,0 +1,212 @@
+---
+id: faqs
+title: Frequently Asked Questions
+sidebar_label: FAQs
+slug: faqs/faqs
+---
+
+
+# FAQs
+
+These are some of the questions we are often asked about using Amplication
+
+
+<details>
+  <summary>What is Amplication?</summary>
+
+  Amplication is an open-source development tool. It helps you develop quality Node.js applications without spending time on repetitive coding tasks. Amplication auto-generates fully functional apps based on TypeScript and Node.js.
+
+</details>
+
+<details>
+  <summary>How does Amplication work?</summary>
+  
+  You just need to define your data models, roles, and permissions, and Amplication will generate a TypeScript Node.js application with everything you need already baked in. This includes fully functional REST API and GraphQL API for all your data models, authentication, role-based authorization, logging, and even an admin UI. 
+
+</details>
+
+
+<details>
+  <summary>Do I need to be an expert developer to work with Amplication?</summary>
+  
+ Amplication is a great tool for developers, whatever their experience.  Even though beginners can get up-and-running quickly with Amplication, you own the code and so can make full use of your code-writing talent to customize yor output and work your magic. 
```

Comment:
Amplication is a great platform for professional developers, whatever their experience.  Even though beginners can get up-and-running quickly with Amplication, you own the code and so can make full use of your code-writing talent to customize your output and work your magic. Amplication is also a great solution for technical leaders who seek boosting their teams' productivity and efficiency.